site/app/models/DockerUI.php

```diff
@@ -20,7 +20,7 @@
  * @method string getLastRan()
  * @method string getSysinfoLastUpdated()
  * @method array getErrorLogs()
- * @method array getDockerImages()
+ * @method array getDockerImages(string $sort = 'name', string $direction = 'ASC')
```

Comment:
remove this line entirely. these should only be added when the magic method is used which you are now overriding